modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -12,18 +12,26 @@ You can activate any of the following feature sets by using the `FeatureGate` CR
 
 * `TechPreviewNoUpgrade`. This featrue set a subset of the current Technology Preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents upgrades. This feature set is not recommended on production clusters. The following Technology Preview features are enabled by this feature set:
 
-** Azure Disk CSI Driver Operator. Enables the provisioning of persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
-** VMware vSphere CSI Driver Operator. Enables the provisioning of persistent volumes (PVs) by using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
-** CSI automatic migration. Enables the automatic migration of supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Available as a Technology Preview for:
+** Persistent storage using the Alibaba AliCloud Disk CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
```

Comment:
This should be removed, here is a typo? that you mentioned both Alibaba AliCloud Disk and Microsoft Azure Disk, but anyway, both of them need to be enabled by feature gate in 4.10. (See https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L113)

Reply:
@duanwei33 is Alibaba in the feature set? I don't see it at https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L113

Reply:
@duanwei33 Can you help with this ^^?

Reply:
@mburke5678 
Really sorry for my typo, none of them need to be enabled by feature gate in 4.10. So this line should be removed.